src/frontend/AppContainerWrapper.tsx

```diff
@@ -21,32 +21,40 @@ import bugsnag from "./lib/logger";
 import { PracticeMode } from "./sharedComponents/PracticeMode";
 import DefaultContainer from "./Navigation/DefaultContainer";
 import OnboardingContainer from "./Navigation/OnboardingContainer";
+import { ExperimentsContainer } from "./Navigation/ExperimentsStack";
 
 // Turn on logging if in debug mode
 if (__DEV__) debug.enable("*");
 const log = debug("mapeo:App");
 
 // WARNING: This needs to change if we change the navigation structure
-const getNavStoreKey = (includeOnboarding: boolean) =>
-  `@MapeoNavigation@${includeOnboarding ? 9 : 8}`;
-
-const createNavigationStatePersister = (includeOnboarding: boolean) => async (
-  navState: NavigationState
-) => {
+//Dev experiments and include onbooarding cannot be turned on at the same time (as discussed with the whole team)
+const getNavStoreKey = (includeOnboarding: boolean, devExperiments: boolean) =>
```

Comment:
Not sure if it will be an issue in reality, but i think this will be hard to maintain if, for example, onboarding-specific navigation changes. In that case, we'd have to increment the onboarding key to be `11` instead of `10` i.e. we'd have to remember to increment the version by 2 instead of 1 because of the variations, and for each variation

I wonder if it would make help to namespace the navigation keys based on the flags so basically starting new keys for the two flags. Thinking of something along these lines:

```ts
// `Standard` is the default navigation. Name can vary however one sees fit
const getNavStoreKey = (variant?: 'DevExperiments' | 'Onboarding') {
  switch (variant) {
    case 'Onboarding':
      // Since this is namespaced, we can start at 1, and then increment this whenever onboarding-specific navigation changes
      return '@MapeoNavigation-Onboarding@1'
    case 'DevExperiments':
         // Since this is namespaced, we can start at 1, and then increment this whenever dev experiments specific navigation changes
      return '@MapeoNavigation-DevExperiments@1'
    default:
      return  '@MapeoNaviagation@8'
   }
  }
}
```

Not sure if there are unforeseen consequences of doing this - any thoughts?

Reply:
Yeah I think i prefer your way of name-spacing here. This does not solve the problem of if we mix these states, but definitely more elegant then nested ternaries.

Reply:
yeah that's true. in theory maybe could combine them so it'd be something like `@MapeoNavigation-Onboarding-DevExperiments@1` but i think that's not necessary to worry about for now 😅

Reply:
I think its clearer now, but im not 100% sure if thats true. I have been staring at this code for too long  😅